bears/general/CopyrightCheckBear.py

```diff
@@ -0,0 +1,30 @@
+import re
+
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Result import Result
+
+
+class CopyrightCheckBear(LocalBear):
+    """
+    Checks for copyright notice in a file.
+    """
+    LANGUAGES = {'All'}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'License'}
+
+    def run(self, filename, file,
+            author: str=''):
+        """
+        :param author: pass the name of the author
+        """
+        copyright_regexp = \
+            r'Copyright\s+(\(C\)\s+)?\d{4}([-,]\d{4})*\s+%(author)s'
+        re_copyright = re.compile(copyright_regexp %
+                                  {'author': author}, re.IGNORECASE)
+        message = ''
+        if not(re_copyright.search(''.join(file))):
+            message = 'Copyright notice not present.'
+
+        yield Result(self, message)
```

Comment:
We are not bound here related to `yeilding/outputting of different and clear message` as it is not any `lint bear` thrn IMO two message template will be great here :grinning: .
one with `Copyright with author specified` 
second with `Copyright with without author `

Reply:
I didn't get this :sweat_smile: as I have added a separate setting for passing author argument if required. So if that argument is not applied, then it will simply yield `copyright notice not found` but if the author is supplied and not found in the file then should this yield `copyright with not specified` author? something like that?

Reply:
yes exactly :+1:

Reply:
accordinly change the test.